src/python/WMCore/WorkQueue/WorkQueue.py

```diff
@@ -12,6 +12,9 @@
 
 from __future__ import division, print_function
 
+from builtins import str as str_, bytes
```

Comment:
I've seen you using `from builtins import str as newstr` in slice 11. Should we adopt the same pattern in here as well?

Reply:
Uh, you are correct, I definitely lacked consistency here! fixed!

Reply:
Pushed new commit with this change